server/api/tumdev/campus_backend.proto

```diff
@@ -649,30 +649,35 @@ message StudyRoom{
   string building_name = 5;
 }
 
-message GetKinoRequest{
+message GetMoviesRequest{
+  // the last Id you have requested. Use -1 to get all movies
   int32 lastId = 1;
 }
 
-message GetKinoReply{
-  repeated KinoMsgElement kinos = 1;
+message GetMoviesReply{
+  repeated MovieMsgElement movies = 1;
 }
 
-message KinoMsgElement{
-  string name = 1;
-  string path = 2;
-  int32 kino = 3;
+message MovieMsgElement{
+  string coverName = 1;
+  string coverPath = 2;
+  int32 id = 3;
   google.protobuf.Timestamp date = 4;
   google.protobuf.Timestamp created = 5;
   string title = 6;
+  // release year of the movie
   string year = 7;
   string runtime = 8;
   string genre = 9;
   string director = 10;
   string actors = 11;
-  string rating = 12;
+  // imdb rating
+  string imdbRating = 12;
```

Comment:
```suggestion
  string imdb_rating = 12;
```

Generally try to use snake case in proto. The code gen then correctly translate that to the respective language standard.

> Use PascalCase (with an initial capital) for message names – for example, SongServerRequest. Use lower_snake_case for field names (including oneof field and extension names) – for example, song_name.

https://protobuf.dev/programming-guides/style/#message-field-names